charts/actions-runner-controller/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v2
+name: actions-runner-controller
+description: A Helm chart for summerwind/actions-runner-controller
+type: application
+
+# chart version
+version: 0.1.0
+
+# app (controller) version
+appVersion: 0.7.2
```

Comment:
it might be good to pull this from `${VERSION}` in the makefile? maybe using another sed statement to substitute it?

Reply:
Done. However, since `make release` is executed in GitHub Actions, this will only affect chart archive in GitHub Releases. The version in the git will remain unchanged.